rmw_connext_cpp/src/identifier.cpp

```diff
@@ -12,9 +12,6 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include "identifier.hpp"
+#include "rmw_connext_cpp/identifier.hpp"
 
-extern "C"
-{
```

Comment:
I don't see a reason to declare the symbol `extern "C"`. Also none of the other RMW impl. does that.